model/testresult/testresult.go

```diff
@@ -106,12 +107,12 @@ func Aggregate(pipeline []bson.M, results interface{}) error {
 }
 
 // TestResultsPipeline is an aggregation pipeline for returning test results to the REST v2 API.
-func TestResultsPipeline(id, filename, status string, limit, sort, execution int) []bson.M {
+func TestResultsPipeline(taskId, testId, status string, limit, sort, execution int) []bson.M {
```

Comment:
this is the worst interface design ever, but I don't think there's much we can do here.